segfast/utils.py

```diff
@@ -35,6 +37,112 @@ def update(self, n=1):
 notifier = Notifier
 
 
+class TraceHeaderSpec:
```

Comment:
Please, add `to_tuple` / `to_dict` method: motivation is to keep track of actually used headers in, for example, `Geometry` class of `seismiqb`.

In the same vain, I would think about returning list of `TraceHeaderSpec` instances out of `Loader.load_headers` methods, so that you have a detailed info on what was actually loaded.

Reply:
The returned specs may differ from those returned by `make_headers_specs`, I'll add such an option in my PR later today